commcare_connect/solicitations/migrations/0008_simplify_response_status.py

```
@@ -0,0 +1,38 @@
+
+
+def reverse_convert_old_statuses(apps, schema_editor):
+    """Reverse migration - not possible to restore original statuses"""
+    # We can't restore the original statuses since we don't know what they were
+    pass
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("solicitations", "0007_remove_completed_status"),
+    ]
```

Ruff (0.14.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)